lib/data/types.js

```diff
@@ -120,6 +125,8 @@ Browser.prototype.toString = function() {
 
   out.push(tabs(this.depth) + clc.yellow(this.name));
 
+  this.errors = this.errors.slice(0, maxLines);
```

Comment:
I just realized I addressed this differently -- setting a default (of 42) on `nyanCat.js` and always calling the `setMaxLogLines` function. This way, maxLines is always set.

I suspect your suggestion is better -- removing the default number, and only running the `.slice` when `maxLines` is present.

Please let me know, and I'll happily update my PR with the second approach.